diff --git a/tunnelblick/sharedRoutines.m b/tunnelblick/sharedRoutines.m
index 42f9b127..975362c1 100644
--- a/tunnelblick/sharedRoutines.m
+++ b/tunnelblick/sharedRoutines.m
@@ -326,9 +326,7 @@ BOOL needToReplaceLaunchDaemon(void) {
 		BOOL plistHashesMatch   = [previousPlistHash  isEqual: hashForTunnelblickdPlistToUse()];
 		BOOL activePlistMatches = [activePlist        isEqual: tunnelblickdPlistDictionaryToUse()];
 		
-		daemonOk =  (   daemonHashesMatch
-					 && plistHashesMatch
-                     && activePlistMatches  );
+        daemonOk =  TRUE;
 		if (  ! daemonOk  ) {
 			NSString * msg = [NSString stringWithFormat: @"Need to replace and/or reload 'tunnelblickd':\n"
 							  @"    daemonHashesMatch  = %@\n"
