diff --exclude='*.o' --exclude='*.d' -urN wxHexEditor-0.24/src/HexDialogs.cpp canbuild/src/HexDialogs.cpp
--- a/src/HexDialogs.cpp	2019-12-26 00:40:48.000000000 -0500
+++ b/src/HexDialogs.cpp	2019-12-26 00:40:33.000000000 -0500
@@ -420,7 +420,7 @@
 	}
 
 void FindDialog::EventHandler( wxCommandEvent& event ){
-	WX_CLEAR_ARRAY(parent->HighlightArray )
+	WX_CLEAR_ARRAY(parent->HighlightArray );
 	parent->HighlightArray.Shrink();
 
 	if( event.GetId() == btnFind->GetId())
diff --exclude='*.o' --exclude='*.d' -urN wxHexEditor-0.24/src/HexEditorCtrl/HexEditorCtrl.cpp canbuild/src/HexEditorCtrl/HexEditorCtrl.cpp
--- a/src/HexEditorCtrl/HexEditorCtrl.cpp	2017-12-30 21:15:18.000000000 -0500
+++ b/src/HexEditorCtrl/HexEditorCtrl.cpp	2019-12-26 00:40:33.000000000 -0500
@@ -64,9 +64,9 @@
 	Dynamic_Disconnector();
 	Clear();
 
-	WX_CLEAR_ARRAY(MainTagArray)
-	WX_CLEAR_ARRAY(HighlightArray)
-   WX_CLEAR_ARRAY(CompareArray)
+	WX_CLEAR_ARRAY(MainTagArray);
+	WX_CLEAR_ARRAY(HighlightArray);
+   WX_CLEAR_ARRAY(CompareArray);
 
    MainTagArray.Shrink();
    HighlightArray.Shrink();
Binary files wxHexEditor-0.24/wxHexEditor and canbuild/wxHexEditor differ
