--- a/plugin/wakatime.vim
+++ b/plugin/wakatime.vim
@@ -447,7 +447,7 @@
                     let pyver = pyver - 1
                 endwhile
                 else
-                    let paths = paths + ['/usr/bin/python3', '/usr/local/bin/python3', '/usr/bin/python3.6', '/usr/local/bin/python3.6', '/usr/bin/python', '/usr/local/bin/python', '/usr/bin/python2', '/usr/local/bin/python2']
+                    let paths = paths + ['/opt/local/libexec/wakatime-python']
                 endif
                 let paths = paths + ['python']
                 let index = 0
