--- configure.ac.old	2020-11-11 05:31:37.000000000 -0500
+++ configure.ac	2020-11-11 05:32:34.000000000 -0500
@@ -8760,10 +8760,6 @@
 # Optionally user can pass an option to configure, i. e.
 # ./configure PYTHON=/usr/bin/python
 # =====================================================================
-if test $_os = Darwin -a "$enable_python" != fully-internal -a "$enable_python" != internal; then
-    # Only allowed choices for macOS are 'internal' (default) and 'fully-internal'
-    enable_python=internal
-fi
 if test "$build_os" != "cygwin" -a "$enable_python" != fully-internal; then
     if test -n "$PYTHON"; then
         PYTHON_FOR_BUILD=$PYTHON
@@ -8833,9 +8829,6 @@
     ;;
 system)
     AC_MSG_RESULT([system])
-    if test "$_os" = Darwin; then
-        AC_MSG_ERROR([--enable-python=system doesn't work on macOS because the version provided is obsolete])
-    fi
     ;;
 *)
     AC_MSG_ERROR([Incorrect --enable-python option])
