diff --git a/libgrive/CMakeLists.txt b/libgrive/CMakeLists.txt
index d9dbebb..ecd85f5 100644
--- a/libgrive/CMakeLists.txt
+++ b/libgrive/CMakeLists.txt
@@ -4,12 +4,10 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 find_package(LibGcrypt REQUIRED)
 find_package(CURL REQUIRED)
-find_package(EXPAT REQUIRED)
 find_package(Boost 1.40.0 COMPONENTS program_options filesystem unit_test_framework regex system REQUIRED)
 find_package(BFD)
 find_package(CppUnit)
 find_package(Iberty)
-find_package(ZLIB)
 
 find_package(PkgConfig)
 pkg_check_modules(YAJL REQUIRED yajl)
@@ -37,10 +35,6 @@ else ( IBERTY_FOUND )
   set( IBERTY_LIBRARY "" )
 endif ( IBERTY_FOUND )
 
-if ( ZLIB_FOUND )
-	set( OPT_LIBS	${OPT_LIBS}	${ZLIB_LIBRARIES} )
-endif ( ZLIB_FOUND )
-
 include_directories(
 	${libgrive_SOURCE_DIR}/src
 	${libgrive_SOURCE_DIR}/test
@@ -88,7 +82,6 @@ target_link_libraries( grive
 	${Boost_REGEX_LIBRARY}
 	${Boost_SYSTEM_LIBRARY}
 	${IBERTY_LIBRARY}
-	${EXPAT_LIBRARY}
 	${OPT_LIBS}
 )
 
diff --git a/libgrive/src/xml/TreeBuilder.cc b/libgrive/src/xml/TreeBuilder.cc
index 9d57501..b2fd5b4 100644
--- a/libgrive/src/xml/TreeBuilder.cc
+++ b/libgrive/src/xml/TreeBuilder.cc
@@ -23,8 +23,6 @@
 #include "Node.hh"
 #include "util/log/Log.hh"
 
-#include <expat.h>
-
 #include <cassert>
 #include <iostream>
 #include <fstream>
