diff --git a/Shifty/Assets.xcassets/edgeIcon.imageset/Contents.json b/Shifty/Assets.xcassets/edgeIcon.imageset/Contents.json
new file mode 100644
index 0000000..3be0603
--- /dev/null
+++ b/Shifty/Assets.xcassets/edgeIcon.imageset/Contents.json
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "edge.png"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
\ No newline at end of file
diff --git a/Shifty/Assets.xcassets/edgeIcon.imageset/edge.png b/Shifty/Assets.xcassets/edgeIcon.imageset/edge.png
new file mode 100644
index 0000000..197bc38
Binary files /dev/null and b/Shifty/Assets.xcassets/edgeIcon.imageset/edge.png differ
diff --git a/Shifty/Base.lproj/Setup.storyboard b/Shifty/Base.lproj/Setup.storyboard
index df80081..283607b 100644
--- a/Shifty/Base.lproj/Setup.storyboard
+++ b/Shifty/Base.lproj/Setup.storyboard
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.Storyboard.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES" initialViewController="dg9-4W-042">
+<document type="com.apple.InterfaceBuilder3.Cocoa.Storyboard.XIB" version="3.0" toolsVersion="15702" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES" initialViewController="dg9-4W-042">
     <dependencies>
         <deployment identifier="macosx"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="14490.70"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="15702"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
@@ -57,7 +57,7 @@
                                 <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="shiftyCircleIcon" id="Csb-ai-sDK"/>
                             </imageView>
                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" verticalCompressionResistancePriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="mxU-q0-bpJ">
-                                <rect key="frame" x="105" y="342" width="265" height="36"/>
+                                <rect key="frame" x="105" y="343" width="265" height="35"/>
                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" alignment="center" title="Welcome to Shifty!" id="9wQ-sP-igZ">
                                     <font key="font" metaFont="systemSemibold" size="30"/>
                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -65,7 +65,7 @@
                                 </textFieldCell>
                             </textField>
                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" verticalCompressionResistancePriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="6jJ-c5-oIY">
-                                <rect key="frame" x="122" y="315" width="232" height="17"/>
+                                <rect key="frame" x="122" y="317" width="232" height="16"/>
                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="Shifty is now running in the menu bar." id="WcY-uu-0Eu">
                                     <font key="font" metaFont="system"/>
                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -73,7 +73,7 @@
                                 </textFieldCell>
                             </textField>
                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" verticalCompressionResistancePriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="ldh-Qb-ijW">
-                                <rect key="frame" x="18" y="71" width="315" height="17"/>
+                                <rect key="frame" x="18" y="71" width="315" height="16"/>
                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="Click Next to continue setup. It'll just take a minute." id="J4B-E9-KIZ">
                                     <font key="font" metaFont="system"/>
                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -97,7 +97,7 @@ DQ
                                 </connections>
                             </button>
                             <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="250" verticalCompressionResistancePriority="250" translatesAutoresizingMaskIntoConstraints="NO" id="jn8-mZ-J20">
-                                <rect key="frame" x="33" y="144" width="410" height="156"/>
+                                <rect key="frame" x="33" y="135" width="410" height="167"/>
                                 <constraints>
                                     <constraint firstAttribute="width" constant="410" id="kBb-Cu-wf7"/>
                                 </constraints>
@@ -142,11 +142,11 @@ DQ
                         <autoresizingMask key="autoresizingMask"/>
                         <subviews>
                             <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="250" verticalCompressionResistancePriority="250" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="mxV-uh-tNL">
-                                <rect key="frame" x="33" y="191" width="410" height="173"/>
+                                <rect key="frame" x="33" y="201" width="410" height="166"/>
                                 <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyUpOrDown" image="websiteShiftingScreenshot-en" id="lih-8C-3IP"/>
                             </imageView>
                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" horizontalCompressionResistancePriority="500" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="sJ8-k4-SBk">
-                                <rect key="frame" x="18" y="372" width="439" height="34"/>
+                                <rect key="frame" x="18" y="375" width="439" height="32"/>
                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="Would you like to let Shifty automatically disable Night Shift when you visit certain websites?" id="hcb-Lm-Zmy">
                                     <font key="font" metaFont="system"/>
                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -154,7 +154,7 @@ DQ
                                 </textFieldCell>
                             </textField>
                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="tfl-ez-xrs">
-                                <rect key="frame" x="18" y="414" width="342" height="36"/>
+                                <rect key="frame" x="18" y="415" width="342" height="35"/>
                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" alignment="left" title="Enable Website Shifting?" id="yjk-M4-AOM">
                                     <font key="font" metaFont="systemSemibold" size="30"/>
                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -202,10 +202,10 @@ DQ
                                 </connections>
                             </button>
                             <customView ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="eYW-HA-94C">
-                                <rect key="frame" x="157" y="64" width="161" height="117"/>
+                                <rect key="frame" x="157" y="51" width="161" height="140"/>
                                 <subviews>
                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="fxw-jc-XI4">
-                                        <rect key="frame" x="-2" y="100" width="165" height="17"/>
+                                        <rect key="frame" x="-2" y="124" width="165" height="16"/>
                                         <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="Website Shifting supports:" id="bNv-dv-CBp">
                                             <font key="font" metaFont="system"/>
                                             <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -213,10 +213,10 @@ DQ
                                         </textFieldCell>
                                     </textField>
                                     <stackView distribution="fill" orientation="vertical" alignment="leading" spacing="4" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="C1T-Yo-Rmw">
-                                        <rect key="frame" x="15" y="0.0" width="146" height="92"/>
+                                        <rect key="frame" x="15" y="0.0" width="146" height="116"/>
                                         <subviews>
                                             <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="IYO-zU-rWM">
-                                                <rect key="frame" x="0.0" y="72" width="146" height="20"/>
+                                                <rect key="frame" x="0.0" y="96" width="146" height="20"/>
                                                 <subviews>
                                                     <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="fC7-e4-xgu">
                                                         <rect key="frame" x="0.0" y="0.0" width="20" height="20"/>
@@ -227,7 +227,7 @@ DQ
                                                         <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="safariIcon" id="ZFb-5U-9xA"/>
                                                     </imageView>
                                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="WWn-Pc-9SO">
-                                                        <rect key="frame" x="26" y="2" width="122" height="17"/>
+                                                        <rect key="frame" x="26" y="2" width="122" height="16"/>
                                                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Safari" id="4ft-mI-wIN">
                                                             <font key="font" metaFont="system"/>
                                                             <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -248,7 +248,7 @@ DQ
                                                 </customSpacing>
                                             </stackView>
                                             <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="9NB-FN-bPk">
-                                                <rect key="frame" x="0.0" y="48" width="146" height="20"/>
+                                                <rect key="frame" x="0.0" y="72" width="146" height="20"/>
                                                 <subviews>
                                                     <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="p8L-eK-sfO">
                                                         <rect key="frame" x="0.0" y="0.0" width="20" height="20"/>
@@ -259,7 +259,7 @@ DQ
                                                         <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="chromeIcon" id="pZL-lW-eR3"/>
                                                     </imageView>
                                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="MmC-i0-VFp">
-                                                        <rect key="frame" x="26" y="2" width="122" height="17"/>
+                                                        <rect key="frame" x="26" y="2" width="122" height="16"/>
                                                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Chrome" id="shv-ze-VLi">
                                                             <font key="font" metaFont="system"/>
                                                             <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -280,7 +280,7 @@ DQ
                                                 </customSpacing>
                                             </stackView>
                                             <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="ntP-Q2-Hwv">
-                                                <rect key="frame" x="0.0" y="24" width="146" height="20"/>
+                                                <rect key="frame" x="0.0" y="48" width="146" height="20"/>
                                                 <subviews>
                                                     <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="5vF-9W-6he">
                                                         <rect key="frame" x="0.0" y="0.0" width="20" height="20"/>
@@ -291,7 +291,7 @@ DQ
                                                         <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="operaIcon" id="gVm-c7-zAp"/>
                                                     </imageView>
                                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="p0g-Ne-ZzW">
-                                                        <rect key="frame" x="26" y="2" width="122" height="17"/>
+                                                        <rect key="frame" x="26" y="2" width="122" height="16"/>
                                                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Opera" id="1Y8-bn-63E">
                                                             <font key="font" metaFont="system"/>
                                                             <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -311,6 +311,38 @@ DQ
                                                     <real value="3.4028234663852886e+38"/>
                                                 </customSpacing>
                                             </stackView>
+                                            <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="LzB-bM-4k7">
+                                                <rect key="frame" x="0.0" y="24" width="59" height="20"/>
+                                                <subviews>
+                                                    <imageView horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="9YB-3T-x9F">
+                                                        <rect key="frame" x="0.0" y="0.0" width="20" height="20"/>
+                                                        <constraints>
+                                                            <constraint firstAttribute="width" constant="20" id="RxX-pp-RMq"/>
+                                                            <constraint firstAttribute="height" constant="20" id="ggr-e8-SJu"/>
+                                                        </constraints>
+                                                        <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="edgeIcon" id="ccf-zt-16I"/>
+                                                    </imageView>
+                                                    <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="5GT-Iq-wnt">
+                                                        <rect key="frame" x="26" y="2" width="35" height="16"/>
+                                                        <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Edge" id="x75-vd-BRJ">
+                                                            <font key="font" metaFont="system"/>
+                                                            <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
+                                                            <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
+                                                        </textFieldCell>
+                                                    </textField>
+                                                </subviews>
+                                                <constraints>
+                                                    <constraint firstItem="5GT-Iq-wnt" firstAttribute="centerY" secondItem="9YB-3T-x9F" secondAttribute="centerY" id="vQK-aF-xNV"/>
+                                                </constraints>
+                                                <visibilityPriorities>
+                                                    <integer value="1000"/>
+                                                    <integer value="1000"/>
+                                                </visibilityPriorities>
+                                                <customSpacing>
+                                                    <real value="3.4028234663852886e+38"/>
+                                                    <real value="3.4028234663852886e+38"/>
+                                                </customSpacing>
+                                            </stackView>
                                             <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="xdg-mR-Qlo">
                                                 <rect key="frame" x="0.0" y="0.0" width="146" height="20"/>
                                                 <subviews>
@@ -323,7 +355,7 @@ DQ
                                                         <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="vivaldiIcon" id="GYy-uD-kEC"/>
                                                     </imageView>
                                                     <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="df5-Zs-JT5">
-                                                        <rect key="frame" x="26" y="2" width="122" height="17"/>
+                                                        <rect key="frame" x="26" y="2" width="122" height="16"/>
                                                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="Vivaldi" id="cXQ-bL-QlF">
                                                             <font key="font" metaFont="system"/>
                                                             <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -359,12 +391,14 @@ DQ
                                             <integer value="1000"/>
                                             <integer value="1000"/>
                                             <integer value="1000"/>
+                                            <integer value="1000"/>
                                         </visibilityPriorities>
                                         <customSpacing>
                                             <real value="3.4028234663852886e+38"/>
                                             <real value="3.4028234663852886e+38"/>
                                             <real value="3.4028234663852886e+38"/>
                                             <real value="3.4028234663852886e+38"/>
+                                            <real value="3.4028234663852886e+38"/>
                                         </customSpacing>
                                     </stackView>
                                 </subviews>
@@ -541,10 +575,10 @@ DQ
                                 <rect key="frame" x="20" y="121" width="435" height="329"/>
                                 <subviews>
                                     <customView translatesAutoresizingMaskIntoConstraints="NO" id="8NH-cV-yLc">
-                                        <rect key="frame" x="0.0" y="87" width="435" height="156"/>
+                                        <rect key="frame" x="0.0" y="95" width="435" height="140"/>
                                         <subviews>
                                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="HOk-mq-u4U">
-                                                <rect key="frame" x="191" y="76" width="54" height="80"/>
+                                                <rect key="frame" x="191" y="74" width="54" height="66"/>
                                                 <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" title="🎉" id="FhS-Jt-Qvm">
                                                     <font key="font" size="50" name="AppleColorEmoji"/>
                                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -552,7 +586,7 @@ DQ
                                                 </textFieldCell>
                                             </textField>
                                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="YeC-1H-Dey">
-                                                <rect key="frame" x="91" y="32" width="253" height="36"/>
+                                                <rect key="frame" x="91" y="31" width="253" height="35"/>
                                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" alignment="center" title="You're good to go!" id="uHP-lQ-mpf">
                                                     <font key="font" metaFont="systemSemibold" size="30"/>
                                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -560,7 +594,7 @@ DQ
                                                 </textFieldCell>
                                             </textField>
                                             <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="qrq-NZ-lPw">
-                                                <rect key="frame" x="134" y="0.0" width="168" height="17"/>
+                                                <rect key="frame" x="134" y="0.0" width="168" height="16"/>
                                                 <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="Thanks for installing Shifty." id="E7p-5d-X2C">
                                                     <font key="font" metaFont="system"/>
                                                     <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
@@ -606,7 +640,7 @@ DQ
                                 </constraints>
                                 <buttonCell key="cell" type="push" title="More info..." bezelStyle="rounded" alignment="center" controlSize="small" borderStyle="border" imageScaling="proportionallyDown" inset="2" id="njq-rZ-B3n">
                                     <behavior key="behavior" pushIn="YES" lightByBackground="YES" lightByGray="YES"/>
-                                    <font key="font" metaFont="smallSystem"/>
+                                    <font key="font" metaFont="menu" size="11"/>
                                 </buttonCell>
                                 <connections>
                                     <action selector="analyticsDetailClicked:" target="j1O-j4-Iou" id="HdD-q6-FZd"/>
@@ -643,6 +677,7 @@ DQ
         <image name="NSGoLeftTemplate" width="9" height="12"/>
         <image name="NSLeftFacingTriangleTemplate" width="9" height="12"/>
         <image name="chromeIcon" width="32" height="32"/>
+        <image name="edgeIcon" width="32" height="32"/>
         <image name="menuBarScreenshot" width="362" height="155.5"/>
         <image name="operaIcon" width="32" height="32"/>
         <image name="safariIcon" width="32" height="32"/>
diff --git a/Shifty/BrowserManager.swift b/Shifty/BrowserManager.swift
index 415de17..bda8f70 100644
--- a/Shifty/BrowserManager.swift
+++ b/Shifty/BrowserManager.swift
@@ -21,6 +21,9 @@ enum SupportedBrowserID: BundleIdentifier {
     case chromeCanary = "com.google.Chrome.canary"
     case chromium = "org.chromium.Chromium"
     
+    case edge = "com.microsoft.edgemac"
+    case edgeBeta = "com.microsoft.edgemac.Beta"
+    
     case opera = "com.operasoftware.Opera"
     case operaBeta = "com.operasoftware.OperaNext"
     case operaDeveloper = "com.operasoftware.OperaDeveloper"
@@ -249,7 +252,7 @@ enum BrowserManager {
         
         let tab: Tab?
         switch browserID {
-        case .chrome, .chromeCanary, .chromium, .opera, .operaBeta, .operaDeveloper, .vivaldi:
+        case .chrome, .chromeCanary, .chromium, .edge, .edgeBeta, .opera, .operaBeta, .operaDeveloper, .vivaldi:
             tab = window.activeTab
         case .safari, .safariTechnologyPreview:
             do {
diff --git a/ShiftyAppcast.xml b/ShiftyAppcast.xml
index c142284..6d6ab10 100755
--- a/ShiftyAppcast.xml
+++ b/ShiftyAppcast.xml
@@ -13,7 +13,7 @@
             ]]>
             </description>
             <pubDate>Sun, 11 Aug 2019 12:30:00 -0400</pubDate>
-            <enclosure url="https://github.com/thompsonate/Shifty/releases/download/1.1.2/Shifty-1.1.2.zip" sparkle:version="63" sparkle:shortVersionString="1.1.2" length="1623481" type="application/octet-stream" sparkle:dsaSignature="MC4CFQCjNjT0/Aeu3fPeTlg2MVL6TgB+awIVAMXUin6ui6ewssERYxvBT7pB3gcU" />
+            <enclosure url="https://github.com/thompsonate/Shifty/releases/download/v1.1.2/Shifty-1.1.2.zip" sparkle:version="63" sparkle:shortVersionString="1.1.2" length="1623481" type="application/octet-stream" sparkle:dsaSignature="MC4CFQCjNjT0/Aeu3fPeTlg2MVL6TgB+awIVAMXUin6ui6ewssERYxvBT7pB3gcU" />
             <sparkle:minimumSystemVersion>10.12.4</sparkle:minimumSystemVersion>
         </item>
         <item>
diff --git a/docs/ShiftyAppcast.xml b/docs/ShiftyAppcast.xml
index c142284..6d6ab10 100755
--- a/docs/ShiftyAppcast.xml
+++ b/docs/ShiftyAppcast.xml
@@ -13,7 +13,7 @@
             ]]>
             </description>
             <pubDate>Sun, 11 Aug 2019 12:30:00 -0400</pubDate>
-            <enclosure url="https://github.com/thompsonate/Shifty/releases/download/1.1.2/Shifty-1.1.2.zip" sparkle:version="63" sparkle:shortVersionString="1.1.2" length="1623481" type="application/octet-stream" sparkle:dsaSignature="MC4CFQCjNjT0/Aeu3fPeTlg2MVL6TgB+awIVAMXUin6ui6ewssERYxvBT7pB3gcU" />
+            <enclosure url="https://github.com/thompsonate/Shifty/releases/download/v1.1.2/Shifty-1.1.2.zip" sparkle:version="63" sparkle:shortVersionString="1.1.2" length="1623481" type="application/octet-stream" sparkle:dsaSignature="MC4CFQCjNjT0/Aeu3fPeTlg2MVL6TgB+awIVAMXUin6ui6ewssERYxvBT7pB3gcU" />
             <sparkle:minimumSystemVersion>10.12.4</sparkle:minimumSystemVersion>
         </item>
         <item>
