diff --git src/calibre/utils/ipc/launch.py src/calibre/utils/ipc/launch.py
index 6d970bfb08..51a21d8110 100644
--- src/calibre/utils/ipc/launch.py
+++ src/calibre/utils/ipc/launch.py
@@ -32,7 +32,7 @@ def renice(niceness):
 
 def macos_edit_book_bundle_path():
     base = os.path.dirname(sys.executables_location)
-    return os.path.join(base, 'ebook-viewer.app/Contents/ebook-edit.app/Contents/MacOS/')
+    return os.path.join(base, 'bin/ebook-edit')
 
 
 def exe_path(exe_name):
@@ -81,7 +81,7 @@ def gui_executable(self):
         if ismacos and not hasattr(sys, 'running_from_setup'):
             if self.job_name == 'ebook-viewer':
                 base = os.path.dirname(sys.executables_location)
-                return os.path.join(base, 'ebook-viewer.app/Contents/MacOS/', self.exe_name)
+                return os.path.join(base, 'bin/ebook-viewer', self.exe_name)
             if self.job_name == 'ebook-edit':
                 return os.path.join(macos_edit_book_bundle_path(), self.exe_name)
 
@@ -149,7 +149,7 @@ def __call__(self, redirect_output=True, cwd=None, priority=None, pass_fds=()):
         If redirect_output is True, output from the child is redirected
         to a file on disk and this method returns the path to that file.
         '''
-        exe = self.gui_executable if self.gui else self.executable
+        exe = self.executable
         env = self.env
         try:
             origwd = cwd or os.path.abspath(os.getcwd())
