diff --git b/extern/CMakeProject-glew.cmake a/extern/CMakeProject-glew.cmake
index 9eca4581de..a08004bda7 100644
--- b/extern/CMakeProject-glew.cmake
+++ a/extern/CMakeProject-glew.cmake
@@ -1,8 +1,6 @@
 if(WITH_SYSTEM_GLEW)
   find_package(GLEW REQUIRED)
   set(GLEW_INCLUDE_DIRS ${GLEW_INCLUDE_DIRS} PARENT_SCOPE)
-  set(GLEW_LIBRARIES ${GLEW_LIBRARIES} PARENT_SCOPE)
-  set(GLEW_LIBRARY_RELEASE ${GLEW_LIBRARY_RELEASE} PARENT_SCOPE)
 else()
 
   set(GLEW_SRC "glew-1.5.8/src/glew.c")
