From e860773799f01d02f281c8c39421e7e34658788d Mon Sep 17 00:00:00 2001
From: Colby Klein <shakesoda@gmail.com>
Date: Sun, 10 Mar 2019 13:11:08 -0700
Subject: [PATCH] fix build: can't noexcept this way on msvc 2013.

---
 src/NoteData.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/NoteData.h b/src/NoteData.h
index 13ba626d90..5f20270ef2 100644
--- a/src/NoteData.h
+++ b/src/NoteData.h
@@ -382,8 +382,7 @@ class NoteData
 /** @brief Allow a quick way to swap notedata. */
 namespace std
 {
-	template<> inline void swap<NoteData>( NoteData &nd1, NoteData &nd2 ) noexcept(is_nothrow_move_constructible<NoteData>::value &&
-      is_nothrow_move_assignable<NoteData>::value) { nd1.swap( nd2 ); }
+	template<> inline void swap<NoteData>( NoteData &nd1, NoteData &nd2 ){ nd1.swap( nd2 ); }
 }
 
 #endif
