diff --git a/src/RageSurface_Load_PNG.cpp b/src/RageSurface_Load_PNG.cpp
index aa4ab5f47f..e3d5079185 100644
--- a/src/RageSurface_Load_PNG.cpp
+++ b/src/RageSurface_Load_PNG.cpp
@@ -12,7 +12,7 @@
 #endif
 #pragma warning(disable: 4611) /* interaction between '_setjmp' and C++ object destruction is non-portable */
 #else
-#include <png.h>
+#include "../extern/libpng/include/png.h"
 #endif // _MSC_VER
 
 namespace
diff --git a/src/RageSurface_Save_PNG.cpp b/src/RageSurface_Save_PNG.cpp
index 25cf308bfe..4bb3bf838f 100644
--- a/src/RageSurface_Save_PNG.cpp
+++ b/src/RageSurface_Save_PNG.cpp
@@ -13,7 +13,7 @@
 #endif
 #pragma warning(disable: 4611) /* interaction between '_setjmp' and C++ object destruction is non-portable */
 #else
-#include <png.h>
+#include "../extern/libpng/include/png.h"
 #endif // _MSC_VER
 
 static void SafePngError( png_struct *pPng, const RString &sStr )
