diff --git NoSleep/NoSleep.xcodeproj/project.pbxproj NoSleep/NoSleep.xcodeproj/project.pbxproj
index 12d9a9e..081e91c 100644
--- NoSleep/NoSleep.xcodeproj/project.pbxproj
+++ NoSleep/NoSleep.xcodeproj/project.pbxproj
@@ -24,10 +24,8 @@
 		9279C75D1BEB29F000DEAB4A /* PreferencesController.m in Sources */ = {isa = PBXBuildFile; fileRef = 9279C75B1BEB29F000DEAB4A /* PreferencesController.m */; };
 		9279C75E1BEB29F000DEAB4A /* PreferencesController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 9279C75C1BEB29F000DEAB4A /* PreferencesController.xib */; };
 		927B4CF21BC7BA9D008199AD /* login.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 927B4CF11BC7BA9D008199AD /* login.framework */; };
-		927B4CF41BC7BB58008199AD /* Sparkle.framework in Copy Frameworks */ = {isa = PBXBuildFile; fileRef = 92A75CE11BC7AE64007DD489 /* Sparkle.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		927FE20719F3C693000CAF7B /* IOKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 927FE20619F3C693000CAF7B /* IOKit.framework */; };
 		927FE20B19F40B06000CAF7B /* Security.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 921A631E1688D45900BCB825 /* Security.framework */; };
-		92A7D8ED1BE33F43008D8575 /* Sparkle.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 92A75CE11BC7AE64007DD489 /* Sparkle.framework */; };
 		92A7D8EE1BE342BD008D8575 /* NoSleep.prefPane in Resources */ = {isa = PBXBuildFile; fileRef = 92B88B6516862BBA00A2993F /* NoSleep.prefPane */; };
 		92B88A7B1685908900A2993F /* NoSleepClientClass.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 92B88A741685908900A2993F /* NoSleepClientClass.cpp */; };
 		92B88A7D1685908900A2993F /* NoSleepExtension.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 92B88A761685908900A2993F /* NoSleepExtension.cpp */; };
@@ -62,7 +60,6 @@
 		92B88BD0168637C400A2993F /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 92B88BCF168637C400A2993F /* main.m */; };
 		92B88BD21686386000A2993F /* NoSleepInterface.m in Sources */ = {isa = PBXBuildFile; fileRef = 92B88AFC1685A6FA00A2993F /* NoSleepInterface.m */; };
 		92B88BD31686386000A2993F /* NoSleepInterfaceWrapper.m in Sources */ = {isa = PBXBuildFile; fileRef = 92B88AFE1685A6FA00A2993F /* NoSleepInterfaceWrapper.m */; };
-		92DAC27A1BC7B45C0069E445 /* Sparkle.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 92A75CE11BC7AE64007DD489 /* Sparkle.framework */; };
 		92E638151BDF5980008C699A /* Version.m in Sources */ = {isa = PBXBuildFile; fileRef = 92E638141BDF5980008C699A /* Version.m */; };
 		92E638161BDF5980008C699A /* Version.m in Sources */ = {isa = PBXBuildFile; fileRef = 92E638141BDF5980008C699A /* Version.m */; };
 /* End PBXBuildFile section */
@@ -157,7 +154,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 10;
 			files = (
-				927B4CF41BC7BB58008199AD /* Sparkle.framework in Copy Frameworks */,
 				923C21571BC7B35300061A6F /* NoSleep.framework in Copy Frameworks */,
 			);
 			name = "Copy Frameworks";
@@ -272,7 +268,6 @@
 				92B88B5E1685B4C400A2993F /* NoSleep.framework in Frameworks */,
 				92B88A90168596BA00A2993F /* Cocoa.framework in Frameworks */,
 				920698861B3C80590075FF17 /* ServiceManagement.framework in Frameworks */,
-				92DAC27A1BC7B45C0069E445 /* Sparkle.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
@@ -291,7 +286,6 @@
 			buildActionMask = 2147483647;
 			files = (
 				921A632D1688F29300BCB825 /* Security.framework in Frameworks */,
-				92A7D8ED1BE33F43008D8575 /* Sparkle.framework in Frameworks */,
 				92B88B9816862D9C00A2993F /* NoSleep.framework in Frameworks */,
 				92B88B6616862BBA00A2993F /* Cocoa.framework in Frameworks */,
 				92B88B6816862BBA00A2993F /* PreferencePanes.framework in Frameworks */,
@@ -591,7 +585,6 @@
 				92B88A89168596BA00A2993F /* Frameworks */,
 				92B88A8A168596BA00A2993F /* Resources */,
 				92B88B351685ABA500A2993F /* Copy Frameworks */,
-				92E109A8225F217000D67184 /* ShellScript */,
 				9249A3401B3B10CF007215D9 /* Copy Executabled */,
 				9249A35C1B3C5B0D007215D9 /* Copy KextHelper */,
 			);
@@ -782,29 +775,6 @@
 		};
 /* End PBXRezBuildPhase section */
 
-/* Begin PBXShellScriptBuildPhase section */
-		92E109A8225F217000D67184 /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 12;
-			files = (
-			);
-			inputFileListPaths = (
-			);
-			inputPaths = (
-				"$(DERIVED_FILE_DIR)/$(PRODUCT_NAME)",
-			);
-			outputFileListPaths = (
-			);
-			outputPaths = (
-				"",
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "LOCATION=\"${BUILT_PRODUCTS_DIR}\"/\"${FRAMEWORKS_FOLDER_PATH}\"\nIDENTITY=\"${EXPANDED_CODE_SIGN_IDENTITY_NAME}\"\ncodesign --verbose --force --deep -o runtime --sign \"$IDENTITY\" \"$LOCATION/Sparkle.framework/Versions/A/Resources/AutoUpdate.app\"\ncodesign --verbose --force -o runtime --sign \"$IDENTITY\" \"$LOCATION/Sparkle.framework/Versions/A\"\n";
-			showEnvVarsInLog = 0;
-		};
-/* End PBXShellScriptBuildPhase section */
-
 /* Begin PBXSourcesBuildPhase section */
 		9249A3321B3AF714007215D9 /* Sources */ = {
 			isa = PBXSourcesBuildPhase;
diff --git NoSleep/NoSleepHelper/MainMenu.xib NoSleep/NoSleepHelper/MainMenu.xib
index 1544945..6f9de78 100644
--- NoSleep/NoSleepHelper/MainMenu.xib
+++ NoSleep/NoSleepHelper/MainMenu.xib
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="8191" systemVersion="15A178w" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19455" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
         <deployment identifier="macosx"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="8191"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19455"/>
     </dependencies>
     <objects>
         <customObject id="-2" userLabel="File's Owner" customClass="NSApplication">
@@ -11,11 +11,10 @@
             </connections>
         </customObject>
         <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
-        <customObject id="-3" userLabel="Application"/>
+        <customObject id="-3" userLabel="Application" customClass="NSObject"/>
         <customObject id="494" customClass="NoSleep_ControlAppDelegate">
             <connections>
                 <outlet property="statusItemMenu" destination="533" id="537"/>
-                <outlet property="updater" destination="574" id="578"/>
             </connections>
         </customObject>
         <customObject id="420" userLabel="Font Manager" customClass="NSFontManager"/>
@@ -34,12 +33,6 @@
                         <action selector="orderFrontStandardAboutPanel:" target="-3" id="571"/>
                     </connections>
                 </menuItem>
-                <menuItem title="Check for Updates..." id="575">
-                    <modifierMask key="keyEquivalentModifierMask"/>
-                    <connections>
-                        <action selector="checkForUpdates:" target="574" id="580"/>
-                    </connections>
-                </menuItem>
                 <menuItem title="Preferences..." id="563">
                     <modifierMask key="keyEquivalentModifierMask"/>
                     <connections>
@@ -63,7 +56,6 @@
             </items>
             <point key="canvasLocation" x="270" y="253.5"/>
         </menu>
-        <customObject id="574" userLabel="SUUpdater" customClass="SUUpdater"/>
         <userDefaultsController representsSharedInstance="YES" id="589"/>
     </objects>
 </document>
diff --git NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.h NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.h
index 5bcd227..808be60 100644
--- NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.h
+++ NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.h
@@ -10,8 +10,6 @@
 
 #import <NoSleep/NoSleepInterfaceWrapper.h>
 
-#import <Sparkle/Sparkle.h>
-
 @class PreferencesController;
 
 @interface NoSleep_ControlAppDelegate : NSObject <NSApplicationDelegate, NoSleepNotificationDelegate> {
@@ -19,8 +17,6 @@
     IBOutlet NSWindow *window;
     IBOutlet NSMenu *statusItemMenu;
     
-    IBOutlet SUUpdater *updater;
-    
     NSStatusItem *statusItem;
     
     NoSleepInterfaceWrapper *noSleep;
@@ -32,8 +28,6 @@
 @property (assign) IBOutlet NSWindow *window;
 @property (assign) IBOutlet NSMenu *statusItemMenu;
 
-@property (assign) IBOutlet SUUpdater *updater;
-
 @property (assign) NSCellStateValue enabled;
 
 @property (retain) PreferencesController *preferences;
diff --git NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
index fd4ee6e..21c759c 100644
--- NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
+++ NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
@@ -21,8 +21,6 @@
 @synthesize window;
 @synthesize statusItemMenu;
 
-@synthesize updater;
-
 static void handleSIGTERM(int signum) {
     if([[((NoSleep_ControlAppDelegate *)[NSApp delegate]) updater] updateInProgress]) {
         return;
diff --git NoSleep/NoSleepPref/NoSleepPreferences.h NoSleep/NoSleepPref/NoSleepPreferences.h
index 8e0ebb1..0d353df 100644
--- NoSleep/NoSleepPref/NoSleepPreferences.h
+++ NoSleep/NoSleepPref/NoSleepPreferences.h
@@ -8,9 +8,8 @@
 
 #import <PreferencePanes/PreferencePanes.h>
 #import <NoSleep/NoSleepInterfaceWrapper.h>
-#import <Sparkle/Sparkle.h>
 
-@interface NoSleepPreferences : NSPreferencePane <NoSleepNotificationDelegate, SUUpdaterDelegate> {
+@interface NoSleepPreferences : NSPreferencePane <NoSleepNotificationDelegate> {
 @private
     NoSleepInterfaceWrapper *m_noSleepInterface;
     BOOL stateAC;
diff --git NoSleep/NoSleepPref/NoSleepPreferences.m NoSleep/NoSleepPref/NoSleepPreferences.m
index 5deff67..dec43f9 100644
--- NoSleep/NoSleepPref/NoSleepPreferences.m
+++ NoSleep/NoSleepPref/NoSleepPreferences.m
@@ -17,14 +17,6 @@ NSBundle *getMainAppBundle() {
     return [NSBundle mainBundle];
 }
 
-- (void)updater:(SUUpdater *)updater didFinishLoadingAppcast:(SUAppcast *)appcast {
-    [self didChangeValueForKey:@"lastUpdateDate"];
-}
-
-- (SUUpdater *)updater {
-    return [SUUpdater updaterForBundle:getMainAppBundle()];
-}
-
 - (IBAction)updateNow:(id)sender {
     [self willChangeValueForKey:@"lastUpdateDate"];
     [[self updater] checkForUpdates:sender];
