From b04da2f9d42f1780d7b93a99b1ca7e437b0846c9 Mon Sep 17 00:00:00 2001
From: Andrew Udvare <audvare@gmail.com>
Date: Wed, 4 Nov 2020 05:03:05 -0500
Subject: [PATCH 4/7] NoSleepPref.xib: fix all constraint issues

---
 .../NoSleepPref/Base.lproj/NoSleepPref.xib    | 87 ++++++++++++++-----
 1 file changed, 67 insertions(+), 20 deletions(-)

diff --git a/NoSleep/NoSleepPref/Base.lproj/NoSleepPref.xib b/NoSleep/NoSleepPref/Base.lproj/NoSleepPref.xib
index 7016e36..05d6ba1 100644
--- a/NoSleep/NoSleepPref/Base.lproj/NoSleepPref.xib
+++ b/NoSleep/NoSleepPref/Base.lproj/NoSleepPref.xib
@@ -26,9 +26,8 @@
                 <rect key="frame" x="0.0" y="0.0" width="668" height="296"/>
                 <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
                 <subviews>
-                    <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="114">
+                    <button translatesAutoresizingMaskIntoConstraints="NO" id="114">
                         <rect key="frame" x="18" y="18" width="264" height="18"/>
-                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
                         <buttonCell key="cell" type="check" title="Start NoSleep utility on system startup" bezelStyle="regularSquare" imagePosition="left" alignment="left" state="on" inset="2" id="115">
                             <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
                             <font key="font" metaFont="system"/>
@@ -42,40 +41,49 @@
                             </binding>
                         </connections>
                     </button>
-                    <imageView fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="166">
+                    <imageView translatesAutoresizingMaskIntoConstraints="NO" id="166">
                         <rect key="frame" x="20" y="219" width="64" height="64"/>
-                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                        <constraints>
+                            <constraint firstAttribute="width" constant="64" id="eCA-U3-8w7"/>
+                        </constraints>
                         <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyUpOrDown" image="Logo" id="167"/>
                     </imageView>
-                    <textField verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="168">
+                    <textField verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" translatesAutoresizingMaskIntoConstraints="NO" id="168">
                         <rect key="frame" x="103" y="234" width="529" height="34"/>
-                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                        <constraints>
+                            <constraint firstAttribute="height" constant="34" id="eV2-Fz-kyZ"/>
+                        </constraints>
                         <textFieldCell key="cell" sendsActionOnEndEditing="YES" title="NoSleep makes closing your MacBook display possible without going to sleep mode." placeholderString="" id="169">
                             <font key="font" metaFont="system"/>
                             <color key="textColor" name="controlTextColor" catalog="System" colorSpace="catalog"/>
                             <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                         </textFieldCell>
                     </textField>
-                    <box autoresizesSubviews="NO" fixedFrame="YES" borderType="line" titlePosition="noTitle" translatesAutoresizingMaskIntoConstraints="NO" id="170">
+                    <box autoresizesSubviews="NO" borderType="line" titlePosition="noTitle" translatesAutoresizingMaskIntoConstraints="NO" id="170">
                         <rect key="frame" x="-32" y="82" width="732" height="124"/>
-                        <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
                         <view key="contentView" id="Egq-GL-AnO">
                             <rect key="frame" x="3" y="3" width="726" height="118"/>
                             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                             <subviews>
-                                <imageView fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="171">
+                                <imageView translatesAutoresizingMaskIntoConstraints="NO" id="171">
                                     <rect key="frame" x="463" y="42" width="64" height="64"/>
-                                    <autoresizingMask key="autoresizingMask" flexibleMinX="YES" flexibleMinY="YES"/>
+                                    <constraints>
+                                        <constraint firstAttribute="width" constant="64" id="lKT-qO-YtV"/>
+                                    </constraints>
                                     <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="AppleBattery" id="178"/>
                                 </imageView>
-                                <imageView fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="172">
+                                <imageView translatesAutoresizingMaskIntoConstraints="NO" id="172">
                                     <rect key="frame" x="202" y="42" width="64" height="64"/>
-                                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                    <constraints>
+                                        <constraint firstAttribute="width" constant="64" id="JUy-z4-zj5"/>
+                                    </constraints>
                                     <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="AppleACW" id="177"/>
                                 </imageView>
-                                <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="173">
+                                <button translatesAutoresizingMaskIntoConstraints="NO" id="173">
                                     <rect key="frame" x="131" y="18" width="192" height="18"/>
-                                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                    <constraints>
+                                        <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="188" id="wX0-IB-xGH"/>
+                                    </constraints>
                                     <buttonCell key="cell" type="check" title="Never sleep on AC Adapter" bezelStyle="regularSquare" imagePosition="left" alignment="left" state="on" inset="2" id="176">
                                         <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
                                         <font key="font" metaFont="system"/>
@@ -84,9 +92,11 @@
                                         <action selector="checkboxEnableACClicked:" target="-2" id="210"/>
                                     </connections>
                                 </button>
-                                <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="174">
+                                <button translatesAutoresizingMaskIntoConstraints="NO" id="174">
                                     <rect key="frame" x="413" y="18" width="164" height="18"/>
-                                    <autoresizingMask key="autoresizingMask" flexibleMinX="YES" flexibleMinY="YES"/>
+                                    <constraints>
+                                        <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="160" id="4G4-zh-QHC"/>
+                                    </constraints>
                                     <buttonCell key="cell" type="check" title="Never sleep on Battery" bezelStyle="regularSquare" imagePosition="left" alignment="left" state="on" inset="2" id="175">
                                         <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
                                         <font key="font" metaFont="system"/>
@@ -96,11 +106,26 @@
                                     </connections>
                                 </button>
                             </subviews>
+                            <constraints>
+                                <constraint firstItem="173" firstAttribute="baseline" secondItem="174" secondAttribute="firstBaseline" id="0T7-rl-q7c"/>
+                                <constraint firstItem="173" firstAttribute="leading" secondItem="Egq-GL-AnO" secondAttribute="leading" constant="133" id="Db9-pK-GU5"/>
+                                <constraint firstItem="174" firstAttribute="baseline" secondItem="173" secondAttribute="firstBaseline" id="Df5-eN-r2P"/>
+                                <constraint firstItem="174" firstAttribute="top" secondItem="171" secondAttribute="bottom" constant="8" symbolic="YES" id="EIM-c6-PcK"/>
+                                <constraint firstItem="174" firstAttribute="leading" secondItem="173" secondAttribute="trailing" constant="94" id="HXO-BE-J3F"/>
+                                <constraint firstItem="172" firstAttribute="leading" secondItem="Egq-GL-AnO" secondAttribute="leading" constant="202" id="SVu-Ga-Nmh"/>
+                                <constraint firstItem="173" firstAttribute="top" secondItem="172" secondAttribute="bottom" constant="8" symbolic="YES" id="UQN-HC-IwL"/>
+                                <constraint firstItem="171" firstAttribute="centerX" secondItem="174" secondAttribute="centerX" id="Ztk-5B-qaJ"/>
+                                <constraint firstItem="172" firstAttribute="top" secondItem="171" secondAttribute="top" id="gDV-MW-7Ru"/>
+                                <constraint firstAttribute="bottom" secondItem="173" secondAttribute="bottom" constant="20" symbolic="YES" id="k0N-dW-ynI"/>
+                                <constraint firstItem="172" firstAttribute="top" secondItem="Egq-GL-AnO" secondAttribute="top" constant="12" id="m7e-pi-jSQ"/>
+                            </constraints>
                         </view>
                     </box>
-                    <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="266">
+                    <button translatesAutoresizingMaskIntoConstraints="NO" id="266">
                         <rect key="frame" x="18" y="45" width="264" height="18"/>
-                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                        <constraints>
+                            <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="260" id="z5j-xc-fxs"/>
+                        </constraints>
                         <buttonCell key="cell" type="check" title="Require password on display opening" bezelStyle="regularSquare" imagePosition="left" alignment="left" continuous="YES" state="on" inset="2" id="267">
                             <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
                             <font key="font" metaFont="system"/>
@@ -114,9 +139,11 @@
                             </binding>
                         </connections>
                     </button>
-                    <button fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="283">
+                    <button translatesAutoresizingMaskIntoConstraints="NO" id="283">
                         <rect key="frame" x="445" y="45" width="205" height="18"/>
-                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                        <constraints>
+                            <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="201" id="81R-a5-6f6"/>
+                        </constraints>
                         <buttonCell key="cell" type="check" title="Use double click to activate" bezelStyle="regularSquare" imagePosition="left" alignment="left" state="on" inset="2" id="284">
                             <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
                             <font key="font" metaFont="system"/>
@@ -126,6 +153,26 @@
                         </connections>
                     </button>
                 </subviews>
+                <constraints>
+                    <constraint firstItem="168" firstAttribute="leading" secondItem="166" secondAttribute="trailing" constant="21" id="7h6-CS-7Cn"/>
+                    <constraint firstItem="170" firstAttribute="top" secondItem="168" secondAttribute="bottom" constant="30" id="8Im-28-nKt"/>
+                    <constraint firstItem="283" firstAttribute="centerX" secondItem="6" secondAttribute="centerX" constant="213.5" id="97X-0A-8zH"/>
+                    <constraint firstItem="166" firstAttribute="leading" secondItem="6" secondAttribute="leading" constant="20" symbolic="YES" id="Fb8-Ck-v8X"/>
+                    <constraint firstItem="266" firstAttribute="top" secondItem="170" secondAttribute="bottom" constant="25" id="OFW-9P-OeF"/>
+                    <constraint firstItem="266" firstAttribute="leading" secondItem="114" secondAttribute="leading" id="U3C-nq-dog"/>
+                    <constraint firstItem="170" firstAttribute="centerX" secondItem="6" secondAttribute="centerX" id="Wtc-e9-ell"/>
+                    <constraint firstItem="266" firstAttribute="trailing" secondItem="114" secondAttribute="trailing" id="XUZ-zs-ubA"/>
+                    <constraint firstAttribute="trailing" secondItem="283" secondAttribute="trailing" constant="20" symbolic="YES" id="bfU-uw-rxv"/>
+                    <constraint firstAttribute="bottom" secondItem="114" secondAttribute="bottom" constant="20" symbolic="YES" id="boM-df-Dqm"/>
+                    <constraint firstItem="166" firstAttribute="centerY" secondItem="168" secondAttribute="centerY" id="emh-I6-t10"/>
+                    <constraint firstItem="266" firstAttribute="baseline" secondItem="283" secondAttribute="baseline" id="iza-3z-yjA"/>
+                    <constraint firstItem="170" firstAttribute="top" secondItem="166" secondAttribute="bottom" constant="15" id="mgL-Jh-IXc"/>
+                    <constraint firstItem="114" firstAttribute="top" secondItem="266" secondAttribute="bottom" constant="13" id="nrm-Z9-CL9"/>
+                    <constraint firstItem="166" firstAttribute="leading" secondItem="266" secondAttribute="leading" id="wa4-Nk-gag"/>
+                    <constraint firstAttribute="trailing" secondItem="168" secondAttribute="trailing" constant="38" id="xUb-GK-PJA"/>
+                    <constraint firstItem="166" firstAttribute="top" secondItem="6" secondAttribute="top" constant="13" id="xoH-lO-FTZ"/>
+                    <constraint firstItem="170" firstAttribute="leading" secondItem="6" secondAttribute="leading" constant="-29" id="zpk-i1-t1K"/>
+                </constraints>
             </view>
             <point key="canvasLocation" x="22" y="121"/>
         </window>
-- 
2.29.2

