From 2b563d0a3304055bf2eb97c069752c5d0c265bfc Mon Sep 17 00:00:00 2001
From: Andrew Udvare <audvare@gmail.com>
Date: Wed, 4 Nov 2020 07:17:49 -0500
Subject: [PATCH 6/7] NoSleepHelper: not loading kext this way

---
 NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m | 2 --
 1 file changed, 2 deletions(-)

diff --git a/NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m b/NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
index 371b73d..196f7e7 100644
--- a/NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
+++ b/NoSleep/NoSleepHelper/NoSleep_ControlAppDelegate.m
@@ -74,8 +74,6 @@ static void handleSIGTERM(int signum) {
     signal(SIGTERM, handleSIGTERM);
     //[updater setUpdateCheckInterval:60*60*24*7];
     
-    [KextLoader loadKext];
-    
     noSleep = [[NoSleepInterfaceWrapper alloc] init];
     if(noSleep == nil) {
         noSleep = [[NoSleepInterfaceWrapper alloc] init];
-- 
2.29.2

