diff --git a/auRevoir/AuRevoir/main.m b/auRevoir/AuRevoir/main.m
index 3968cfa..cfe7be6 100644
--- a/auRevoir/AuRevoir/main.m
+++ b/auRevoir/AuRevoir/main.m
@@ -23,7 +23,7 @@ int main(int argc, const char * argv[])
     osVersion = [[NSProcessInfo processInfo] operatingSystemVersion];
     
     //only support h sierra
-    if(osVersion.minorVersion != 13)
+    if(osVersion.minorVersion < 13)
     {
         //init alert
         alert = [[NSAlert alloc] init];
